app/views/public_pages/home.html.erb

```diff
@@ -1,5 +1,5 @@
-<% content_for :page_title do%>
-  <%=t("views.public_pages.home.title") %>
+<% content_for :page_title do %>
```

Comment:
A lot of these changes are just whitespace changes as a result of reformatting the file with cmd+opt+L